Library/Koala/Koala.swift

```diff
@@ -1723,18 +1727,19 @@ private func properties(user user: User, prefix: String = "user_") -> [String:An
   return properties.prefixedKeys(prefix)
 }
 
-private func properties(userActivity userActivity: NSUserActivity) -> [String:AnyObject] {
-  let properties: [String: AnyObject?] = [
-    "user_activity_type": userActivity.activityType,
-    "user_activity_title": userActivity.title,
-    "user_activity_webpage_url": userActivity.webpageURL?.absoluteString,
-    "user_activity_keywords": Array(userActivity.keywords),
-  ]
-  return properties.compact()
```

Comment:
a lot of stuff like the above (i.e. creating inline dictionaries of `[String:AnyObject?]` and then compacting) has been replaced by a mutable dictionary `[String:Any]` and setting keys one-by-one.

this is again due to the preference of `Any` over `AnyObject`.